sdk/aqueduct/utils/serialization.py

```diff
@@ -114,7 +114,7 @@ def _read_local_bytes_content(path: str) -> bytes:
 def _read_local_tf_keras_model(path: str) -> Any:
     from tensorflow import keras
 
-    return keras.saving.load_model(path)
+    return keras.models.load_model(path)
```

Comment:
If this is backwards compatible (works with multiple TF versions), do we need to pin the TF version in the dockerfile?

Reply:
Lower bounded it at the latest TF release.

Reply:
I think we'll want to just run the parameter operators locally as a long-term solution for K8s.